client/app/components/packages/landuse-form/project-area.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class PackagesLanduseFormProjectAreaComponent extends Component {
+  // The following four properties represent lists of fields
+  // dependent on the value of a respective radio group.
+  // For a radio group property (such as dcpEntiretycommunity),
+  // the associated list of fields are those which should be
+  // cleared (reset) when the dcpEntiretycommunity value changes.
+  dcpNotaxblock = [
+    'dcpSitedatapropertydescription',
+  ]
+
+  dcpEntiretycommunity = [
+    ...this.dcpNotaxblock,
+    'dcpNotaxblock',
+    'dcpCommunity',
+  ];
+
+  dcpEntiretyboroughs = [
+    ...this.dcpEntiretycommunity,
+    'dcpEntiretycommunity',
+    'dcpBoroughs',
+  ];
+
+  dcpWholecity = [
+    ...this.dcpEntiretyboroughs,
+    'dcpEntiretyboroughs',
+  ];
+
+  get landuseForm() {
+    return this.args.form.data;
+  }
+
+  @action
+  resetDependentFields(fields) {
+    fields.forEach((field) => {
+      this.landuseForm[field] = null;
```

Comment:
Is there a way to reset the changeset? I'm guessing we don't expose access to changeset methods, but this might be relevant:

https://github.com/poteto/ember-changeset#rollbackproperty

Reply:
oh yeah, I honestly forgot that `form.data`  is a changeset, maybe because since it's so well abstracted.
Great idea to use the changeset api. I think using `.set()` here would be good. 
Looks like "rollback" will just roll it back to the previous value instead of clearing the value. 
That will have unintended consequences. 

For example,
  1. saved value of an input field is "Douglas street"
  2. user changes it to "Court street"
  3. user toggles a parent radio group
  4. the input field changes back to "Douglas street"